lib/CL/pocl_util.c

```diff
@@ -2244,13 +2244,14 @@ pocl_update_event_finished (cl_int status, const char *func, unsigned line,
   if (ops->notify_event_finished)
     ops->notify_event_finished (event);
   POCL_UNLOCK_OBJ (event);
-  POname (clReleaseEvent) (event);
```

Comment:
Interestingly the "free event" test failed on the Windows bot. Could there be something wrong here?

Reply:
Well, it didn't fail on a rerun which hints to a race.